templates/master/00-master/alibabacloud/units/openshift-alibabacloud-routes.service.yaml

```diff
@@ -0,0 +1,11 @@
+name: openshift-alibabacloud-routes.service
+enabled: false
```

Comment:
I know this is copied over from the azure/gcp code but could you remind me why this is disabled by default? Or does it not matter since the path service triggers it?